Code/Framework/AzCore/AzCore/Script/ScriptContext.cpp

```diff
@@ -4334,15 +4365,7 @@ LUA_API const Node* lua_getDummyNode()
                 }
 
                 lua_atpanic(m_lua, &LuaPanic);
-
-                //if (modules & ScriptContext::MODULE_DEFAULT)
-                //{
-                luaL_openlibs(m_lua);
-                //}
-                //else
-                //{
-                //AZ_Assert(false, "Modules not supported!");
-                //}
+                OpenLibsAndRemoveLoadLibFunction(m_lua);
```

Comment:
This is a bit overly specific, perhaps `ConfigureLuaLibraries` or `ManagedOpenLuaLibraries` ?

Reply:
Fair enough